src/app/ir/mainNodes.ts

```diff
@@ -0,0 +1,29 @@
+import { randomUuid, UUID } from '@/app/util';
+
+type MLNode = ModelNode
```

Comment:
Why do we need MLNode? And should be `MlNode` to be camelCase compliant

Reply:
sure revising

Reply:
we need it because eventually we will have more kinds than just a `ModelNode` - then `MlNode` will become a union type